uri.opam

```diff
@@ -15,21 +15,16 @@ tags: [
   "org:mirage"
   "org:xapi-project"
 ]
-build: [ "ocaml" "pkg/pkg.ml" "build" "--pinned" "%{pinned}%" "--tests" "false" ]
-build-test: [
-  [ "ocaml" "pkg/pkg.ml" "build" "--pinned" "%{pinned}%" "--tests" "true" ]
-  [ "ocaml" "pkg/pkg.ml" "test" ]
-]
+build: [ "jbuilder" "build" "@install" ]
+build-test: [ "jbuilder" "build" "@runtest" ]
 depends: [
   "base-bytes"
-  "ocamlbuild" {build}
   "ocamlfind" {build}
+  "jbuilder" {build}
   "ounit" {test & >= "1.0.2"}
-  "ppx_deriving"
```

Comment:
It's not needed any more in jbuilder, since it supports an alternative method of generating the ppx driver binary that is more independent of the specific OCaml version used.  See https://github.com/janestreet/ppx_type_conv/issues/7 for the explanation.